client/src/components/LoginPage.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+
+	<div class="login">
+
+		<div class="g-signin2" ref="g-signin2" data-onsuccess="onSignIn"></div>
+
+		<!-- <button @click="signOut">Sign out</button> -->
+
+	</div>
+
+</template>
+
+<script>
+
+import { mapActions } from 'vuex';
+
+export default {
+	name: 'login-page',
+
+	mounted() {
+
+		const self = this;
+
+		window.onSignIn = (googleUser) => {
+
+			const idToken = googleUser.getAuthResponse().id_token;
+
+			self.authenticate({ idToken });
+
+		};
+
+	},
+
+	methods: {
+
+		...mapActions([
+
+			'authenticate',
+			// 'signOut',
+
+		]),
+	},
+};
+</script>
+
+<!-- Add "scoped" attribute to limit CSS to this component only -->
+<style scoped>
```

Comment:
Je pense qu'en vertu du principe (de précaution) YAGNI (You Aint Gonna Need It), il vaut mieux effectivement le retire. Si on n'en n'a pas besoin maintenant, on ne le met pas maintenant. C'est considéré comme du "stock". Donc un "potentiel gaspillage" au sens Lean Develoment. Ça alourdit le code — "moins y a de code, mieux on se porte" — et la charge cognitive pour le réflecteur.